models/trip.go

```diff
@@ -1,28 +1,48 @@
 package models
 
-import "errors"
+import (
+	"github.com/satori/go.uuid"
+)
 
 // A trip represents a one-time visit to a particular country
 type Trip struct {
 	Id string `json:"id"`
 	User string `json:"user"`
-	Place string `json:"place"`
+	Country string `json:"country"`
```

Comment:
Agree. Everything you see here is very WIP at the moment. Our idea was to use an external API for countries, but I'm not sure yet which options we would have